### DISCUSSION TITLE
VS2019 failed to use static libraries

### DISCUSSION THREAD

**Discussion by wuwao1** [ImageMagick/ImageMagick#3445] [Help] [answered]
I tried to use VS2019 to compile the source code[ link](https://imagemagick.org/script/install-source.php#windows), then use the dynamic library in the program, the program runs normally.Then I used the static library and the program failed to compile. The error is as follows
LINK : fatal error LNK1104: cannot open file 'CORE_DB_jxl_.lib'
However, I can't find this lib. There is no static library after source compilation
![image](https://user-images.githubusercontent.com/29857592/112453925-1fe8ee00-8d93-11eb-9599-c5084c5847bb.png)

**Comment by dlemstra** [comment, marked as answer]
Thanks for reporting this. This should be resolved in the next release. As a temporary work around you can apply the changes of this commit: https://github.com/ImageMagick/ImageMagick/commit/6e672a331586775c92d8fc280c6e873ba3b4038e.

**Comment by Bxaa** [comment]
+
